.zuul.yaml

```diff
@@ -1,8 +1,82 @@
 ---
+- nodeset:
+    name: stf-crc_extracted-ocp412
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2.19.0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp413
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2.28.0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp414
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2.30.0-xxl
+
+- job:
+    name: stf-base-2node
+    parent: podified-multinode-edpm-deployment-crc
+    abstract: true
+    required-projects:
+      - name: openstack-k8s-operators/dataplane-operator
+        override-checkout: main
+      - name: openstack-k8s-operators/infra-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-operator
```

Comment:
This one says github.com but the others do not. Is it that either string works and this is just a benign inconsistency?

Reply:
In general, either works as long as there is just one provider for the repo.
i.e. if there were two projects defined with the same name (maybe if we're following a fork as well, or upstream/downstream) we'd always have to use the fully qualified name.


I'll update this to use the fully qualified name, since that's clearer and more consistent.